pkg/virt-api/api.go

```diff
@@ -204,6 +205,7 @@ func subresourceAPIGroup() metav1.APIGroup {
 func (app *virtAPIApp) composeSubresources(ctx context.Context) {
 
 	subresourcesvmGVR := schema.GroupVersionResource{Group: v1.SubresourceGroupVersion.Group, Version: v1.SubresourceGroupVersion.Version, Resource: "virtualmachines"}
+	//versionGVR := schema.GroupVersionResource{Group: v1.GroupVersion.Group, Version: v1.GroupVersion.Version, Resource: "version"}
```

Comment:
you probably didn't mean to leave this in

Reply:
Yes my bad